### DISCUSSION TITLE
process stuck at recv_multipart() for hours

### DISCUSSION THREAD

**Discussion by RenYuanXue** [pycaret/pycaret#2142] [Q&A] [answered]
from pycaret.datasets import get_data
boston = get_data('boston')
from pycaret.regression import *
exp_name = setup(data = boston,  target = 'medv')

The code I wrote and the code from tutorial all stucks at executing the function recv_multipart(), since the status shows "preprocessing data", I tried adding the parameter "preprocess = False". However, it didnt help speeding up the process.

So what does the function recv_multipart() do and what can I do to speed up my execution time?

**Comment by ngupta23** [comment, marked as answer]
Can you try on Google Colab to confirm that you are facing the same issue? Want to eliminate your local machine-specific issues.

**Reply by RenYuanXue** [reply]
Hi I think I found the problem. Once I set the parameter "silence" to True, the code is working perfectly fine.
